### PR TITLE
10x generic ogre emblems

### PR DESCRIPTION
**Pull request by kaldrskegg**
generic ogre emblems

## Changelog:
Added Ogre Coat of Arms

## Developer changelog:

- Added ce_ogre1.dds - ce_ogre12.dds to gfx\coat_of_arms\colored_emblems
- Added ce_ogreempire.dds, the highmaul emblem from Warlords of Draenor
- Assigned ce_ogreempire.dds to e_gorian_empire in 00_wc_landed_titles.txt

### CODE REVIEW

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt [hunk 1]
```diff
@@ -411,4 +411,13 @@ k_tranquillien = {
...
+}
+
+### Ogre ###
+
+e_gorian_empire = {
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 60 0 60 }
+	colored_emblem = {
+		texture = "ce_ogreempire.dds"
+		color1 = hsv360 { 33 96 91 }
```
[the remark this answers is not in the document]
Comment (kaldrskegg) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:423]: I dont know what this means?

[the remark this answers is not in the document]
Comment (zumbak04) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:423]: > I dont know what this means?

This file ends with this and doesn't have closing brackets.

[the remark this answers is not in the document]
Comment (kaldrskegg) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:423]: fixed

**Review by runtd — APPROVED** [review]
All of the icons look nice (even zoomed out) and seem to be lore-friendly. Approved.

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt [hunk 2]
```diff
@@ -700,3 +700,13 @@ d_darkspear = {
 		color3 = rgb { 255 231 167 }
```
Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:700]: Sometimes I wonder why you don't expand the symbols to the max, to the borders.
<details>
<summary>Click to expand</summary>

image
image
image
image

</details>
P.S. There are many symbols like this.

Comment (zumbak04) [inline common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:700]: Many CoAs should be a bit higher, closer to the center I represented by a red dot.
<details>
<summary>Click to expand</summary>

image
image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt (hunk 2 above)
Comment (kaldrskegg) [reply common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt:700]: Try now

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by StarsPotato — APPROVED** [review]
Looks very cool :ogre:

**Pull request merged**